DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1

```diff
@@ -319,10 +365,7 @@ function Set-TargetResource
     }
     else
     {
-        if($AcceptSelfSignedCertificates -and ($AcceptSelfSignedCertificates -eq $false))
```

Comment:
The If block will never be true so the code would never run.
I think I made a mistake here as I intended to comment out the line so it would be brought up to discuss the purpose.
I will change the If block to trigger only when $AcceptSelfSignedCertificates is $false

Reply:
I had second thoughts on making this change. 
Wouldn't fixing the If statement result in a breaking change?
Would it be better to put it back how it was and add a comment for it to be checked if it's still needed?